docs/using-nativeaot/compiling.md

```diff
@@ -0,0 +1,76 @@
+# Compiling with Native AOT
+
+This document explains how to compile and publish your project using Native AOT toolchain. First, please _ensure that [pre-requisites](prerequisites.md) are installed_. If you are starting a new project, you may find the [HelloWorld sample](../../samples/HelloWorld/README.md) directions useful.
+
+## Add ILCompiler package reference
+
+To use Native AOT with your project, you need to add a reference to the ILCompiler NuGet package containing the Native AOT compiler and runtime. Make sure the `nuget.config` file for your project contains the following package sources under the `<packageSources>` element:
+```xml
+<add key="dotnet-experimental" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json" />
+<add key="nuget" value="https://api.nuget.org/v3/index.json" />
+```
+
+If your project has no `nuget.config` file, it may be created by running
+```bash
+> dotnet new nuget
+```
+
+from the project's root directory. New package sources must be added after the `<clear />` element if you decide to keep it.
```

Comment:
```suggestion
from the project's root directory.
```
I do not see a good reason for why this has to be after the `<clear />` element.

Reply:
`dotnet new nuget` creates the `nuget.config` file with the `<clear />` element, which means "ignore the records above". If you add new package sources before the `<clear />` element, they will be ignored as I just confirmed. So you have to either remove the `<clear />` element (thus inheriting the global NuGet package sources) or, if you decide to keep `<clear />`, add new package sources after it.